Directory.Build.targets

```diff
@@ -12,6 +12,8 @@
   <PropertyGroup Condition="'$(UseBootstrapLayout)' == 'true'">
     <!-- Keep in sync with outputs defined in Microsoft.NETCore.Platforms.csproj. -->
     <BundledRuntimeIdentifierGraphFile>$([MSBuild]::NormalizePath('$(BootstrapRidGraphDir)', 'runtime.json'))</BundledRuntimeIdentifierGraphFile>
+    <!-- If our bootstrap build didn't add a new RID to the graph, fall back to the frozen graph. -->
+    <BundledRuntimeIdentifierGraphFile Condition="!Exists('$(BundledRuntimeIdentifierGraphFile)')">$([MSBuild]::NormalizePath('$(LibrariesProjectRoot)', 'Microsoft.NETCore.Platforms', 'src', 'runtime.json'))</BundledRuntimeIdentifierGraphFile>
```

Comment:
I think we should fallback to `PortableRuntimeIdentifierGraph.json` as it is default since .NET 8.

Reply:
I want to keep this consistent with the non-bootstrapping path below.

Reply:
It will likely be removed in the future, since there is no new distro version added in past few years, so it's just a legacy baggage at this point. The distro graph comes from SB and portable graph is used in official builds. Since .NET 10 is going to use VMR for official builds, we may want to double check. (not for this PR)

Reply:
The runtime repo needs to build with it as long as the SDK supports the fallback. When we remove it from the repo as a whole as can adjust it here.